cabal-install-solver/src/Distribution/Solver/Modular/Validate.hs

```diff
@@ -90,7 +90,7 @@ import Distribution.Types.PkgconfigVersionRange
 data ValidateState = VS {
   supportedExt        :: Extension -> Bool,
   supportedLang       :: Language  -> Bool,
-  presentPkgs         :: PkgconfigName -> PkgconfigVersionRange  -> Bool,
+  presentPkgs         :: PkgconfigName -> PkgconfigVersionRange  -> Maybe Bool,
```

Comment:
Would this type work? I think this is closer to the structure of the errors.

```haskell
Maybe (PkgconfigName -> PkgconfigVersionRange  -> Bool)
```

Reply:
Indeed! good point